src/bindings/python/src/openvino/opset1/ops.py

```diff
@@ -9,7 +9,7 @@
 from functools import partial
 
 from openvino.runtime import Node, PartialShape, Type
-from openvino.runtime.op import Constant, Parameter, tensor_iterator
+from openvino.op import Constant, Parameter, tensor_iterator
```

Comment:
why Ops like Parameter are located w/o version namespace ? In C++ it's v0

If we introduce v20.Parameter, how to understand what parameter is used in this file?

Reply:
Let's discuss it in a new task. Here I want to move `opsets` to new location without introducing new logic